modules/ocf_hpc/manifests/compute.pp

```diff
@@ -2,11 +2,12 @@
   require ocf_hpc
 
   # install proprietary nvidia drivers and CUDA.
-  ocf::repackage {
-    ['nvidia-driver', 'nvidia-settings']:
+  ocf::repackage { ['nvidia-driver', 'nvidia-settings', 'nvidia-cuda-toolkit']:
       backport_on => stretch;
+  } -> file { '/etc/modules-load.d/nvidia-uvm.conf':
+    # The nvidia-uvm kernel module, which is needed for CUDA apps, can't be loaded as needed from within Singularity.
+    content => 'nvidia-uvm\n';
```

Comment:
This needs to be a double quoted string, or else `\n` won't be interpreted as an escape sequence.